AdobeUi/view/adminhtml/web/js/components/grid/column/overlay.js

```diff
@@ -25,6 +25,15 @@ define([
             }
 
             return {};
+        },
+
+        isLicensed: function (record) {
```

Comment:
Please extend overlay from AdobeUi module in AdobeStockImageAdminUi module and introduce logic related to Adobe Stock there (same for js file and template).